module4/persistence/src/main/java/com/epam/esm/entity/Role.java

```diff
@@ -0,0 +1,29 @@
+package com.epam.esm.entity;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import java.util.Set;
+
+@Entity
+@Data
+@ToString(exclude = "users")
+@EqualsAndHashCode(exclude = "users")
+public class Role {
```

Comment:
Could be implemented as plain collection in User. See https://docs.jboss.org/hibernate/orm/5.4/userguide/html_single/Hibernate_User_Guide.html#collections